src/components/tooltip/tooltip.spec.ts

```diff
@@ -387,6 +387,21 @@ describe('Tooltip', () => {
       await hideComplete();
       expect(tooltip.open).to.be.false;
     });
+
+    it('should switch role to "status" when `sticky` is true, and back to "tooltip" when false', async () => {
+      expect(tooltip.sticky).to.be.false;
+      tooltip.sticky = true;
+      await elementUpdated(tooltip);
+
+      let internalsRole = (tooltip as any)._internals.role;
+      expect(internalsRole).to.equal('status');
+
+      tooltip.sticky = false;
+      await elementUpdated(tooltip);
+
+      internalsRole = (tooltip as any)._internals.role;
```

Comment:
Don't directly test the internal implementation as that may change because of refactors, additional features, etc.

If you need this test, then create a shown sticky tooltip and run the a11y checks against it.